Automattic/sensei
# Support for mobile email clients

Resolves #6687 
Resolves #6690 

## Proposed Changes
* Don't use styles from the theme
* Fix the gmail media query issue. 
* Move the email styles to the assets folder and use Sass
* Fix the site title alignment (centralized with the logo)
* Fix the Powered by alignment (centralized with the image)
* Fix multiple issues related to margin and padding
* Fix the issue when there is no site logo available.

## Testing Instructions 
Litums link to the Welcome e-mail [here](https://litmus.com/checklist/public/q0Wf5WfWHhsQC3Tq).

1. Run the email recreation tool.
2. Make sure any other change was stored on the Sensei e-mail page template.
3. Go to each template and click on "preview" 
4. Check if the header and footer are there.
5. Send the emails or

* Go to the preview page
* Copy the iFrame HTML
* Get a free account on the site [Litmus](https://www.litmus.com/) - Credit Card required
* Go to the home https://litmus.com/home
* Click "Create new"
* Click Build
* Select the "Blank e-mail"  option
* Paste the email HTML.
* Click "Email previews" (Warning, you need to select only few clients because the trial version is limited to 500 previews).
* You can see the welcome email [here](https://litmus.com/checklist/public/q0Wf5WfWHhsQC3Tq) 





### Screenshots

**Gmail Android**
<img width="368" alt="Screenshot 2023-03-27 at 21 49 13" src="https://user-images.githubusercontent.com/38718/228063358-312ef3be-154d-4ffb-86d1-e2bd43adc3c1.png">

**Gmail iOS**
<img width="402" alt="Screenshot 2023-03-27 at 21 48 46" src="https://user-images.githubusercontent.com/38718/228063362-d9c49bcf-7c53-4348-86fb-be1c128a0741.png">


## Pre-Merge Checklist
